src/SFA.DAS.AANHub.Application.UnitTests/SFA.DAS.AANHub.Application.UnitTests.csproj

```diff
@@ -8,6 +8,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Profiles\Queries\GetProfilesQueryResultTests.cs" />
```

Comment:
Not sure what this is about?

Reply:
You need to delete the test file which is not required.

Reply:
Deleted.